Riot/Modules/Authentication/Views/TermsView.swift

```diff
@@ -0,0 +1,208 @@
+/*
+ Copyright 2018 New Vector Ltd
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import Reusable
+
+final class TermsView: UIView, NibOwnerLoadable, UITableViewDelegate, UITableViewDataSource {
+
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var acceptButton: UIButton!
+
+    @objc weak var delegate: MXKAuthInputsViewDelegate?
+
+    private var acceptedCallback: (()->Void)?
+
+
+    /// NavigationVC to display a policy content
+    private var navigationController: RiotNavigationController?
+
+    /// The list of policies to be accepted by the end user
+    private var policies: [MXLoginPolicyData] = []
+
+    /// Policies already accepted by the end user
+    /// Combined with `policies`, this is the view model for `tableView`
+    private var acceptedPolicies: Set<Int> = []
+
+    /// The index of the policy being displayed fullscreen within `navigationController`
+    private var displayedPolicyIndex: Int?
+
+
+    // MARK: - Setup
+
+    convenience init() {
+        self.init(frame: CGRect.zero)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        loadNibContent()
+        commonInit()
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        loadNibContent()
+        commonInit()
+    }
+
+    private func commonInit() {
+
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.separatorStyle = .none
+        tableView.register(TableViewCellWithCheckBoxAndLabel.nib(), forCellReuseIdentifier: TableViewCellWithCheckBoxAndLabel.defaultReuseIdentifier())
+
+        acceptButton.layer.cornerRadius = 5
```

Comment:
Update corner radius in `layoutSubviews` to be sure frame is set correctly before modifying layer